migration/migrator/migrations/course/20210610100802_course_materials_json_to_db.py

```diff
@@ -0,0 +1,160 @@
+                        'type': material_type,
+                        'release_date': itemvalue['release_datetime'],
+                        'hidden_from_students': itemvalue['hide_from_students'],
+                        'priority': itemvalue['sort_priority']
+                    }
+                    result = database.session.execute(query, params)
+                    course_material_id = result.fetchone()[0]
+                    for section in sections:
+                        query = """
+                            INSERT INTO course_materials_sections (
+                                course_material_id,
+                                section_id
+                            )
+                            VALUES (
+                                :course_material_id, :section_id
+                            ) ON CONFLICT(course_material_id, section_id) DO NOTHING
+                            """
+                        params = {
+                            'course_material_id': course_material_id,
+                            'section_id': section
+                        }
+                        database.session.execute(query, params)
+                    subpath = path[len(str(course_materials_dir))+1:]
+                    dirs = subpath.split('/')
+                    dirs.pop()
+                    curpath = str(course_materials_dir)
+                    for dir in dirs:
+                        curpath += '/' + dir
+                        paths.add(curpath)
+            for dir in paths:
+                query = """
+                    INSERT INTO course_materials (
+                        type,
+                        path,
+                        release_date,
+                        hidden_from_students,
+                        priority
+                    )
+                    VALUES (
+                        :type, :path, :release_date, :hidden_from_students, :priority
+                    ) ON CONFLICT(path) DO NOTHING
+                """
+                params = {
+                    'path': dir,
+                    'type': 2,
+                    'release_date': None,
+                    'hidden_from_students': None,
+                    'priority': 0
+                }
+                database.session.execute(query, params)
+
+
+
+def down(config, database, semester, course):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("DROP TABLE IF EXISTS course_materials CASCADE")
+    database.execute("DROP TABLE IF EXISTS course_materials_sections CASCADE")
```

Comment:
Is this meant to be changed to not drop the tables? I'm confused as to the agreed upon direction of this migration.